modules/inspecting-pod-and-container-logs.adoc

```diff
@@ -37,7 +37,7 @@ Logs retrieved using the preceding `oc logs` commands are composed of messages s
 +
 [source,terminal]
 ----
-$ oc exec <pod_name> ls -alh /var/log
+$ oc exec <pod_name> ls /var/log
```

Comment:
@mburke5678 i see command below works fine but with a small change, can be command be updated as below ? Thanks !!
oc exec <pod_name> -- ls -alh /var/log

Reply:
@kasturinarra Thank you for your help. This is the output I am seeing. Is this the expected result of this command?
```
$ oc exec hello-node-6fbccf8d9-74n96 -- ls -alh /var/log
total 0      
drwxr-xr-x    2 root     root           6 Apr  4  2022 .
drwxr-xr-x    1 root     root          17 Apr  4  2022 ..
```